src/modules/addlikeapi.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-unused-expressions */
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/oXAYSzFtiLQq1yK2hMWK/likes';
+
+// const addApiLikes = (id) => {
+//   fetch(baseUrl, {
+//     method: 'POST',
+//     headers: { 'Content-type': 'application/json; charset=UTF-8' },
+//     body: JSON.stringify({
+//       item_id: id,
+//     }),
+//   });
+// };
```

Comment:
- [OPTIONAL] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)